website/blog/why-we-are-writing-sdks.md

```diff
@@ -0,0 +1,38 @@
+---
+date: "06/11/2021"
+title: "Why We Are Writing SDKs"
+author: Mike Elsmore
+author_url: "https://github.com/ukmadlz"
+author_image_url: "/img/team/mike-elsmore.jpg"
+category: Community
+---
+
+I’m new around here, and my first post is about something I really love… code that makes life easier!
+
+To begin with Optics CLI is ridiculously easy to start using within your application, you simply head to the root of your directory and run `api init`, configure your `.optic.yml` to wrap your application start command, and tada you’ve started gathering data about your ÅPI endpoints by just using them. Super simple and great right? To look at the full process and implement it yourself, please check out [our documentation](https://www.useoptic.com/docs/).
+
+But what happens when you can’t do that? Plenty of instances exist where it’s very complex to work in a team and wrap the command that starts your application. What happens when you use feature flags on that command, or you have to make operating system specific changes?
+
+<!-- truncate -->
+
+## Introducing Optic SDKs
+Firstly, the idea behind these isn’t to be able to interact with Optics Cloud, or with the learning process of the CLI tool, it’s to simplify capturing the API traffic and sending the captures to the CLI. So when looking at it you won’t see handy API classes etc to interact with.
```

Comment:
so essentially we're calling them SDKs for Optic's developer tooling, with an implementation for each API framework

Reply:
And SDK per language, which ends up as a middleware for the frameworks yes.